### PR TITLE
Update enviroment to no longer search for cases using respondent part…

### PR DESCRIPTION
**Pull request by gemmai95**
# Motivation and Context

This change is required to update case events in the case service and for the acceptance tests to run without BI cases.

# What has changed

Stopped wait for case to update method checking for cases using respondent party id.

# How to test?

Tested with changes in case, casesvc-api, frontstage and party.

# Links

Trello:
https://trello.com/c/V8twXPlu

PRs:
https://github.com/ONSdigital/rm-casesvc-api/pull/30
https://github.com/ONSdigital/rm-case-service/pull/70
https://github.com/ONSdigital/ras-party/pull/153
https://github.com/ONSdigital/rm-action-service/pull/79
https://github.com/ONSdigital/rm-collection-exercise-service/pull/108

### CODE REVIEW

File: acceptance_tests/features/steps/display_trading_as_external.py [hunk 1]
```diff
@@ -34,11 +33,10 @@ def respondent_has_completed_survey(_):
...
     # Complete the survey by phone
     ru_ref = '49900000007'
-    party_id = get_party_by_email('example@example.com')['id']
-    wait_for_ru_specific_cases_to_update(party_id, ru_ref)
+    business_party = get_party_by_ru_ref(ru_ref)
     ce_id = get_collection_exercise('cb8accda-6118-4d3b-85a3-149e28960c54', '201801')['id']
-    update_case_group_status(ce_id, ru_ref, 'COMPLETED_BY_PHONE')
-
+    case = get_b_case(ce_id, business_party['id'])
+    post_case_event(case['id'], 'COMPLETED_BY_PHONE', 'Completed by phone')
```
Comment (andrewmil) [inline acceptance_tests/features/steps/display_trading_as_external.py]: :+1:

File: controllers/case_controller.py [hunk 2]
```diff
@@ -18,22 +18,22 @@ def get_case_by_id(case_id):
...
-    logger.debug('Posting case event', case_id=case_id, party_id=party_id, category=category)
+def post_case_event(case_id, category, description):
+    logger.debug('Posting case event', case_id=case_id, category=category)
 
     url = f'{Config.CASE_SERVICE}/cases/{case_id}/events'
     payload = {
         'description': description,
         'category': category,
-        'partyId': party_id,
+        'partyId': None,
```
Comment (andrewmil) [inline controllers/case_controller.py]: We should just remove the `partyId` field entirely here I think

File: run.py [hunk 3]
```diff
@@ -6,4 +6,4 @@
     format = os.getenv('BEHAVE_FORMAT', 'progress2')
 
     behave_executable.main(args=f'--format {format} system_tests/features')
-    behave_executable.main(args=f'--format {format} acceptance_tests/features')
+    behave_executable.main(args=f'--format {format} acceptance_tests/features/notification_letter.feature')
```
Comment (andrewmil) [inline run.py]: Is this right?

Comment (gemmai95) [reply run.py]: Nope. That was when I was trying to fix the notification feature

File: run.py [hunk 4]
```diff
@@ -6,4 +6,4 @@
     format = os.getenv('BEHAVE_FORMAT', 'progress2')
 
     behave_executable.main(args=f'--format {format} system_tests/features')
-    behave_executable.main(args=f'--format {format} acceptance_tests/features')
+    behave_executable.main(args=f'--format {format} acceptance_tests/features/enrolment-status-disable.feature')
```
Comment (andrewmil) [inline run.py]: why is this just running one feature?

**Pull request merged**